tasks/bash.sh

```diff
@@ -53,6 +72,15 @@ if [ -z "${name}" ]; then
     release=$(uname -r)
 fi
 
+# puppet_agent install task can ask for each of these values
+if test "x$1" = "xplatform"; then
```

Comment:
Can we add a test that these work? Maybe using run_script?

Reply:
Yes, Probably a good time to switch over to test helpers in BoltSpec anyway. Working on it.